packages/ui/src/components/popups/DatePickerPopup.svelte

```diff
@@ -0,0 +1,160 @@
+<!--
+// Copyright © 2020 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { IntlString } from '@anticrm/platform'
+  import Label from '../Label.svelte'
+  import Back from '../icons/Back.svelte'
+  import Forward from '../icons/Forward.svelte'
+  import { createEventDispatcher } from 'svelte'
+
+  export let label: IntlString
+  export let value: Date = new Date(Date.now())
+
+  let view: Date = value
+  const months: Array<string> = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December'
+  ]
+  let monthYear: string
+  let days: Array<number>
+
+  const dispatch = createEventDispatcher()
+
+  const daysInMonth = (date: Date): number => {
+    return 33 - new Date(date.getFullYear(), date.getMonth(), 33).getDate()
+  }
+
+  $: {
+    monthYear = months[view.getMonth()] + ' ' + view.getFullYear()
+    days = []
+    for (let i = 1; i <= daysInMonth(view); i++) {
+      days.push(new Date(view.getFullYear(), view.getMonth(), i).getDay())
+    }
+  }
+</script>
+
+<div class="datepicker-popup">
+  {#if label}<div class="title"><Label {label} /></div>{/if}
+  <div class="flex-between">
+    <button
+      class="arrow"
+      on:click|preventDefault={() => {
+        view.setMonth(view.getMonth() - 1)
+        view = view
+      }}><Back size={16} /></button
+    >
+    <div class="caption-color">
+      {monthYear}
+    </div>
+    <button
+      class="arrow"
+      on:click|preventDefault={() => {
+        view.setMonth(view.getMonth() + 1)
+        view = view
+      }}><Forward size={16} /></button
+    >
+  </div>
+  <div class="calendar">
+    <div class="flex-center caption">Mo</div>
```

Comment:
we need to use internationalisation routines to obtain all of this information instead of predefined ones.

Reply:
@SasLord let's create an issue to handle it as next step.